generators/java/sdk/src/main/java/com/fern/java/client/generators/WrappedRequestGenerator.java

```diff
@@ -144,6 +173,10 @@ public GeneratedWrappedRequest generateFile() {
                         .map(objectProperty ->
                                 generatedObject.objectPropertyGetters().get(objectProperty))
                         .collect(Collectors.toList()))
+                .addAllPathParams(pathParameterObjectProperties.stream()
```

Comment:
nit: similar to above w.r.t. property order.

Reply:
done